src/new/wallet.js

```diff
@@ -230,6 +230,69 @@ class HathorWallet extends EventEmitter {
     }
   }
 
+  getAddressInfo(address, options = {}) {
+    storage.setStore(this.store);
+    const { token = HATHOR_TOKEN_CONFIG.uid } = options;
+
+    // Throws an error if the address does not belong to this wallet
+    if (!this.isAddressMine(address)) {
+      throw new Error('Address does not belong to this wallet.');
+    }
+
+    // Derivation path index
+    const index = this.getAllAddresses().indexOf(address);
```

Comment:
It will consume a lot of memory if the address list is big. I feel we should get the index in a different way.

Reply:
You can use `this.getAddressIndex(address)`

Reply:
I switched to `this.getAddressIndex(address)`, thanks.